### PR TITLE
Ddf 1559: Remove Cardinality Workarounds

### PR DESCRIPTION
**Pull request by figliold**
@andrewkfiedler @garrettfreibott @oconnormi @lessarderic

[Review on Reviewable](https://reviewable.io/reviews/codice/ddf/334)

### CODE REVIEW

**Comment by oconnormi** [comment]
<sup>**[libs/common-system/pom.xml, line 34 \[r2\]](https://reviewable.io:443/reviews/codice/ddf/334#-K2w_wmLn6uykL6r75Mk)** ([raw file](https://github.com/codice/ddf/blob/c14ecb361fdad576747760bdb776077a3c9cc527/libs/common-system/pom.xml#L34)):</sup>
We should look at updating the hamcrest version in dependency management to 1.3 instead of 1.1 at some point


---


Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/codice/ddf/334)

**Comment by oconnormi** [comment]
<sup>**[distribution/ddf-common/src/main/resources/etc/DDF_Custom_Mime_Type_Resolver.csw.config, line 5 \[r2\]](https://reviewable.io:443/reviews/codice/ddf/334#-K2wbxPr-IzTI5YofEq7)** ([raw file](https://github.com/codice/ddf/blob/c14ecb361fdad576747760bdb776077a3c9cc527/distribution/ddf-common/src/main/resources/etc/DDF_Custom_Mime_Type_Resolver.csw.config#L5)):</sup>
Is it okay that the file name is `DDF_Custom_Mime_Type_Resolver.csw.config` even though in the features file the config was called `DDF_Custom_Mime_Type_Resolver-CswCustomMimeTypes`?


---


Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/codice/ddf/334)

**Comment by oconnormi** [comment]
We should probably squash these commits at some point


---







Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/codice/ddf/334)

**Comment by oconnormi** [comment]
Reviewed 7 of 13 files at r1, 7 of 15 files at r2.
Review status: 14 of 25 files reviewed at latest revision, 2 unresolved discussions.

---



Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/codice/ddf/334)

File: platform/security/sts/security-sts-anonymousclaimshandler/src/main/resources/OSGI-INF/blueprint/blueprint.xml [hunk 1]
```diff
@@ -25,8 +25,12 @@
         <cm:managed-properties persistent-id="ddf.security.sts.anonymousclaims"
                                update-strategy="container-managed"/>
         <!-- Default properties -->
-        <property name="attributes"
-                  value="http://schemas.xmlsoap.org/ws/2005/05/identity/claims/nameidentifier=anonymous,http://schemas.xmlsoap.org/ws/2005/05/identity/claims/role=anonymous"/>
+        <property name="attributes">
+            <array value-type="java.lang.String">
+                <value>http://schemas.xmlsoap.org/ws/2005/05/identity/claims/nameidentifier=anonymous</value>
```
Comment (stustison) [inline platform/security/sts/security-sts-anonymousclaimshandler/src/main/resources/OSGI-INF/blueprint/blueprint.xml]: This change is going to collide with https://github.com/codice/ddf/pull/331 so depending on which one goes in first, this is going to need to be fixed.

**Comment by figliold** [comment]
Review status: 14 of 25 files reviewed at latest revision, 3 unresolved discussions.

---

<sup>**[distribution/ddf-common/src/main/resources/etc/DDF_Custom_Mime_Type_Resolver.csw.config, line 5 \[r2\]](https://reviewable.io:443/reviews/codice/ddf/334#-K2wbxPr-IzTI5YofEq7)** ([raw file](https://github.com/codice/ddf/blob/c14ecb361fdad576747760bdb776077a3c9cc527/distribution/ddf-common/src/main/resources/etc/DDF_Custom_Mime_Type_Resolver.csw.config#L5)):</sup>
Yes, tested and it works.


---


Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/codice/ddf/334)

**Comment by oconnormi** [comment]
Reviewed 4 of 13 files at r1, 7 of 15 files at r2.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---



Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/codice/ddf/334)

**Pull request merged**